tensorflow/lite/toco/tflite/operator_test.cc

```diff
@@ -158,6 +158,37 @@ TEST_F(OperatorTest, SimpleOperators) {
   CheckSimpleOperator<TensorFlowRankOperator>("RANK", OperatorType::kRank);
 }
 
+TEST_F(OperatorTest, BuiltinAbs) {
+  AbsOperator abs_op;
+  abs_op.inputs = {"input"};
+  auto operator_by_type_map = BuildOperatorByTypeMap(false /*enable_flex_ops*/);
+  const BaseOperator* op = operator_by_type_map.at(abs_op.type).get();
+
+  Model float_model;
+  Array& input_float_array = float_model.GetOrCreateArray(abs_op.inputs[0]);
+  input_float_array.data_type = ArrayDataType::kFloat;
+  OperatorSignature float_signature = {.op = &abs_op, .model = &float_model};
```

Comment:
unfortunately this .op notation breaks on come compilers :(

Reply:
Ok then.. I was copying from some other tests...